### ISSUE TITLE
Enabling moon content configuration breaks all moon-specific custom scrap spawning.

### ISSUE THREAD

**Issue by Teacyn** [IAmBatby/LethalLevelLoader#109]
I've verified this pretty extensively at this point. Here is a testing profile that contains nothing but Gratar and Etern from Wesley's moons (alongside dependencies), both of which spawn exclusively custom scrap. Gratar has customization enabled (but no changes to scrap), Etern does not.
`018fd914-3844-af5a-6599-e6171c2fe93f`
Going in-game will show that Gratar is now spawning none of the custom scrap. Etern spawns scrap fine. It seems like the game or LLL overrides the custom scrap with default scrap, as V-type engines and metal sheets will begin to spawn on Gratar.

I have confirmed that custom scrap from other sources does still spawn on customized moons. An easy way to verify this is to install ImmersiveCompany, as the screwdrivers and gemstones from it will spawn in addition to whatever default scrap the game seems to use. Additionally, moons that use base game scrap are unaffected. Enabling customization for a moon from StarlancerMoons for example, the scrap it has defined all still spawns correctly. The issue seems to be confined exclusively to custom scrap defined by a custom moon.

**Comment by Gamersxp6248** [comment]
Can confirm issue occurs with any custom moon with custom scraps. It is impossible to define custom scrap per moon.

**Comment by ZD-ChaofanJ** [comment]
Yeah, this is still an issue as of LethalLeveLLoader v1.3.7. Anything related to Custom Scrap seems to completely break if you have "Enable Content Configuration" set to true for any moon, Vanilla or otherwise. I tried having it only spawn "Lava Crystal" from Magic_Wesley's Infernis but instead, the scrap just doesn't show up at all. I tried specifying scrap from my own scrap mod (Jacob's Scrap Pack), which depends on LethalLib, and instead of spawning the specified scrap, it just spawned only one specific  (and wrong) type of scrap. I haven't tested custom enemies but from looking at the comments above, I doubt those work properly either.

When it comes to Content Configuration for Vanilla Scrap and Enemies, it seems to work just fine but if you specify anything custom, it just doesn't seem to work. Which... Partially defeats the purpose of the "Content Configuration" feature.

I wanted to try adding custom moons to the game and my own modpack but for me, it's not really worth it if you can't control spawn rates properly, outside of vanilla assets.

I'm assuming this is a bug, but regardless, LethalLevelLoader needs to be more "Custom Content Friendly". Currently, the only way to control custom scrap spawn rates for custom moons is with LethalLib, and that's only if you are the mod creator, and isn't really do-able for the standard mod user/player.

To replicate the issue;
1. Install any mod featuring Custom Scrap (this includes mods like Magic_Wesley's moon mods).
2. Run the game, enter any file, then close the game (this builds the configs, assuming you are running a new install/profile).
3. Enable Content Configuration for your moon of choice.
4. Customize the Scrap Spawn rate of said moon of choice to a custom scrap item (e.g. Lava Crystal:100).
5. Run the game and enter the moon of choice that you configured content configuration for.
6. Look around the interior for scrap and you will notice either incorrect or missing scrap.
7. Optionally, install Imperium and check the object/item list, and you will notice the same thing, without needing to go inside the interior.

**Comment by Teacyn** [comment]
I have a further update on this myself.

It appears that this issue is caused by something with Wesley's moons specifically. Just tested it with Seichi, another moon with custom scrap, and it works fine. Saw a report in the discord server that apparently Wesley has the scrap on his moons defined/registered incorrectly and intends to fix it with the v60 update. 

It appears that this issue is a side effect of that error on Wesley's end, as such this is likely not a bug with LLL at all. Though, it may still be possible for LLL to detect this error and either warn the modder or fix it automatically?

Can anyone confirm it happening on any moons with custom scrap that are not Wesley's?
